app/views/admin/admin_users/show.html.erb

```diff
@@ -4,7 +4,9 @@
   <!--/descriptions-->
 
   <!--subnav-->
+  <p class="note">To fix common errors with this user's Subscriptions and Stats pages, and to reindex their works and bookmarks, choose "Troubleshoot Account."<p>
```

Comment:
This needs to be `</p>` at the end. Also, I think this should be in <%= ts() %>?